VTEX - Subscriptions API (v3).json

```diff
@@ -2283,7 +2634,7 @@
                         "description": "Time reference in which subscription orders will be created.",
```

Comment:
Não entendi esse array. São as frequências respectivamente correspondentes às periodicidades possíveis do plano?

Pode ser só minha ignorância em relação ao produto, então veja se vale a pena mudar ou não.

Reply:
São os dias de compra e variam de acordo com a periodicidade. Se a periodicidade for semanal, o dia de compra pode ser todo domingo (`Sunday`) se a periodicidade for mensal pode ser sempre em julho (`7`). Optei por não fazer um "enum" porque considerei que ficaria mais confuso. Ficou mais claro?